stuff/pipelines/test_contest_queries.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+'''test verification_code module.'''
+
+import datetime
+import os
+import random
+import string
+import sys
+import time
+
+import omegaup.api
+
+import credentials
+import database.contest
+import test_constants
+
+
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "."))
+import lib.db   # pylint: disable=wrong-import-position
+
+
+def test_get_contest_contestants() -> None:
+    '''Test get contest contestants'''
+
+    dbconn = lib.db.connect(
+        lib.db.DatabaseConnectionArguments(
+            user=credentials.MYSQL_USER,
+            password=credentials.MYSQL_PASSWORD,
+            host=credentials.MYSQL_HOST,
+            database=credentials.MYSQL_DATABASE,
+            port=credentials.MYSQL_PORT,
+            mysql_config_file=lib.db.default_config_file_path() or ''
+        )
+    )
+    with dbconn.cursor(buffered=True, dictionary=True) as cur:
+        contests = database.contest.get_contests(
+            cur=cur,
+            date_lower_limit=test_constants.DATE_LOWER_LIMIT,
+            date_upper_limit=test_constants.DATE_UPPER_LIMIT,
+        )
+        number_of_contests = len(contests)
+
+    client = omegaup.api.Client(api_token=test_constants.API_TOKEN,
+                                url=test_constants.OMEGAUP_API_ENDPOINT)
+    current_time = datetime.datetime.now()
+    future_time = datetime.datetime.now() + datetime.timedelta(hours=5)
```

Comment:
```suggestion
    future_time = current_time + datetime.timedelta(hours=5)
```

ningún método debe obtener la hora más de una vez porque eso causa flakiness.

Reply:
Listo